### PR TITLE
Updated ReadMe with my Sprint report

### PR DESCRIPTION
**Pull request by Tyler-Ouellette**
Can we make sure I did this right, still learning most of the git commands with merges and branches and etc.

### CODE REVIEW

File: docs/group/README.md [hunk 1]
```diff
@@ -28,16 +28,65 @@
...
+#### Tyler Ouellette
+
+<p>
+1. I created a Trello board for the group that is being used for our weekly sprints. 
+2. I installed composer for mac and laravel. I had some issues with the set up process. I created a document with all the instructions of what I did and followed of the laracast videos and included screen shots of the errors that I had. 
+3. After looking over the process and re-doing it, I got it working properly.  The issue was within my zshrc file Path.
+4. Uploaded my documentation guide to the github.
+5. Created a review of the laracasts videos with basic examples and documentation for quick reference. Will be uploading the documentation once formatting is complete.
+6. Watched the Laracast Videos up to video 10.
+
```
Comment (James-N-M) [inline docs/group/README.md]: close <p> tag

File: docs/group/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Group
+s# Group
```
Comment (James-N-M) [inline docs/group/README.md]: remove s

File: docs/group/README.md [hunk 3]
```diff
@@ -28,16 +28,65 @@
...
+
+#### Sahil Verma
+<p>
+1.    Installation of Laravel on local machine complete - the installation process involved downloading a composer, PHP and Laravel. The process wasn’t a smooth and simple installation as some steps needed proper attention. In the PC documentation guide I created, I listed all the steps that need to be completed in-order for a smooth installation. Example, making sure the latest version of PHP was installed so the composer could work, disabling certain extensions in the php.ini file so Laravel could install properly etc. If followed properly, Laravel will install and you will be able to work with the file in an editor of your choice. 
+2.    Documentation Guide posted on GitHub (formatting will be adjusted for easier reading)
+3.    Downloaded local repository that James has posted in GitHub to keep in-line with group for our web application What’s For Dinner
+4.    Watching Laracast videos/YouTube tutorials to familiarize myself with the environment and extra practice for when we start coding!
+</p>
+
+#### Tyler Ouellette
```
Comment (James-N-M) [inline docs/group/README.md]: good job

**Comment by James-N-M** [comment]
@Tyler-Ouellette left a couple comments, but what ever code editor your using is formatting code differently then mine, hense the green changes to everything lol. Well have to look into a style ci thinggy so we all have the same thing

File: docs/group/README.md (hunk 2 above)
Comment (Tyler-Ouellette) [reply docs/group/README.md]: good catch, pretty sure happened when switching from my prompt to vscode

**Pull request merged**